hack/generator/pkg/codegen/pipeline/status_augment.go

```diff
@@ -113,6 +113,14 @@ func flattenAugmenter(allTypes astmodel.ReadonlyTypes) augmenter {
 						changed = true
 					}
 
+					if len(swaggerProp.Description()) > len(specProp.Description()) {
```

Comment:
if this is in here we should rename the flattenAugmenter to something like propertyAugmenter and pull out the bits that copy the flatten property and copy the documentation into an easily-extensible set